common/config/rollup/rollup.config.js

```diff
@@ -17,5 +18,5 @@ export default (packageJson) => ({
     format: 'cjs',
     sourcemap: true
   },
-  plugins: [commonjs(), sourcemaps(), svg()]
+  plugins: [commonjs(), sourcemaps(), svg(), json({ compact: true })]
```

Comment:
Hmm worried this will bundle all strings for all languages into the js file, not sure how we would avoid this - I'll file a bug

Reply:
https://skype.visualstudio.com/SPOOL/_workitems/edit/2495870